### ISSUE TITLE
 CMIP7 Standard names: lake_area_fraction

### ISSUE THREAD

**Issue by christian-stepanek** [cf-convention/vocabularies#222]
**Christian Stepanek**

**2024-09-12**

For each term please try to give the following:

**- Proposed term to appear in the vocabulary**  lake_area_fraction

**- Description:** This is similar to land_area_fraction etc. but shall describe the fraction of lake coverage on a grid cell. It appears that there is not yet a name for the fraction of a grid cell covered by lakes. This is relevant for, e.g., PMIP, where differences in lake cover must be quantified.

**- Unit:** Fraction

**Comment by taylor13** [comment]
There is related (duplicate?) discussion in https://github.com/cf-convention/vocabularies/issues/65 .  I think somewhere we've decided that the way to handle specific area type fractions in the future is through adding that area type to the area type table and then attaching a scalar dimension to the area fraction variable.  It's standard name would simply be "area_fraction".  Will try to fill in more details later.

**Comment by martinjuckes** [comment]
I agree with Karl, in CMIP6 we added many area fractions in this way (e.g. for bare soil, grass, C3 grass, C4 grass, broadleaf deciduous trees, etc).

We would need a new `lake` term in [the area type table](https://cfconventions.org/Data/area-type-table/current/build/area-type-table.html)

**Comment by efisher008** [comment]
Dear all, 

Thank you Christian @christian-stepanek for your proposal. As @taylor13 and @martinjuckes have responded, this would ideally be handled by a new `lake` term in the area type table as has been mentioned in the discussion in issue #65 and utilising the dimensionality of the area fraction variable to achieve this. Do you have further questions on this? It sounds as though @taylor13 is happy to give some further clarification around how this would work if so. 

Best wishes,
Ellie

**Comment by japamment** [comment]
Dear Christian @christian-stepanek,

I agree with @taylor13 and @martinjuckes that the best way forward is to add `lake` as a new entry to the [CF area type table](https://cfconventions.org/Data/area-type-table/current/build/area-type-table.html). Your variable could then use the existing standard name area_fraction and you would need to add a scalar coordinate variable with the standard name of area_type whose value would be the string 'lake'.

Would you be okay with taking this approach?

Best wishes,
Alison

**Comment by christian-stepanek** [comment]
This sounds like a sensible approach @japamment @taylor13 @martinjuckes, thanks a lot. I have to see how the details are to be implemented into the CMIP7 airtable framework, but it sounds like this has been done before.

All the best
Christian

**Comment by japamment** [comment]
Dear Christian, @christian-stepanek 

Thanks for your response. It seems then that we are agreed on the general approach of adding a new area type.

Karl @taylor13 pointed out that there is a parallel discussion in [#65](https://github.com/cf-convention/vocabularies/issues/65) where a similar area type was proposed by @gnikulin:
`lake_and_inland_sea`
**Description:** An area type of lake_and_inland_sea means a body of water surrounded by land.

This seems to be agreed in #65. @christian-stepanek would you be okay with also using this? It will be added to the area type table in the next update (planned for January 20th-21st).

Best wishes,
Alison

**Comment by christian-stepanek** [comment]
Sorry for delayed reply during my paternity leave. Yes, I am fine with the description lake_and_inland_sea.

**Comment by japamment** [comment]
@christian-stepanek

Many thanks for your response. There has been some further discussion in [#65](https://github.com/cf-convention/vocabularies/issues/65) and consensus has now been reached on the following: 

`lake_and_inland_sea`
**Description:** The area type lake_and_inland_sea means a quasi-permanent body of water surrounded by land, which may include certain ponds and reservoirs, but excludes rivers and streams which by gravity convey water into and out of lakes or inland seas or into the ocean.

[#65](https://github.com/cf-convention/vocabularies/issues/65) will be accepted in seven days unless further comments are received in that issue. This discussion here will therefore also be marked as accepted in seven days unless further comments are received.

Best wishes,
Alison

**Comment by christian-stepanek** [comment]
@japamment 

Thanks a lot for your help with this. All this sounds fine to me. As soon as the standard name has been accepted I would update respective variables in our CMIP7 data request.

All the best
Christian

**Comment by japamment** [comment]
@christian-stepanek I am pleased to say that `lake_and_inland_sea` has now been accepted in [#65](https://github.com/cf-convention/vocabularies/issues/65) so I am  marking this issue as "Accepted" too. The term will be published in the next update of the area type table (planned for 17th/18th March).

Best wishes,
Alison

**Comment by japamment** [comment]
The change agreed here was published in [V13](https://cfconventions.org/Data/area-type-table/13/build/area-type-table.html) of the CF area type table so closing this issue as complete.
